packages/runtime/src/widgets/Form/Date/Date.tsx

```diff
@@ -85,28 +85,32 @@ export const Date: React.FC<DateProps> = (props) => {
   }, [value, formInstance]);
 
   return (
-    <EnsembleFormItem
-      initialValue={
-        values?.initialValue ? dayjs(values.initialValue) : undefined
-      }
-      values={values}
-    >
-      <DatePicker
-        disabled={values?.enabled === false}
-        disabledDate={(d): boolean =>
-          (Boolean(values?.lastDate) && d.isAfter(values?.lastDate)) ||
-          (Boolean(values?.firstDate) && d.isBefore(values?.firstDate))
-        }
-        format={values?.format}
-        onChange={onDateChange}
-        placeholder={values?.hintText}
-        style={{ width: "100%", ...values?.styles }}
-        value={value}
-        {...(values?.showCalendarIcon === false
-          ? { suffixIcon: false }
-          : undefined)}
-      />
-    </EnsembleFormItem>
+    <>
+      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
+      <div onClick={(e) => e.stopPropagation()}>
```

Comment:
This will be necessary for all custom elements in the dropdown if manual option is set to true, so you should add this wrapper inside the Dropdown not to each individual widget.